content/projekte/automatischer-rolladen/contents.lr

```diff
@@ -0,0 +1,27 @@
+title: automatischer Rolladen
+---
+author: L3D
+---
+body:
+
+Was viele nicht wissen: In der Toolbox gibt es an einigen Fenstern Automatischer Rolladen.
+
+Dieser lässt sich durch eine hochkomplexe und ausgeklügelte Drehbewegung an einem runden dreh-schalter steuern.
+Unerwartet hierbei ist auch, dass das betätigen des Schalters nicht über Alexa, Google Home oder Siri funktioniert, sondern über physische interaktion, oft auch als "cloudless" oder "offline" bezeichnet, funktioniert.
+
+Für den Fall das eine Veränderung des Zustandes eines der Rolladen gewünscht ist, wird um die selbstständige Betätigung eines solches Schalters gebeten!
+---
+icon: fab fa-windows
+---
+pub_date: 2019-02-16
+---
+teaser: Was viele nicht wissen: In der Toolbox gibt es an einigen Fenstern Automatischer Rolladen.
```

Comment:
teaser: Was viele nicht wissen: In der Toolbox gibt es an einigen Fenstern automatische Rollläden.